tests/core_validation/test_known_argument_names.py

```diff
@@ -0,0 +1,135 @@
+import pytest
+
+from graphql.core.language.location import SourceLocation
+from graphql.core.validation.rules import KnownArgumentNames, KnownDirectives
+from utils import expect_passes_rule, expect_fails_rule
+
+
+def unknown_arg(arg_name, field_name, type_name, line, column):
+    return {
+        'message': KnownArgumentNames.message(arg_name, field_name, type_name),
+        'locations': [SourceLocation(line, column)]
+    }
+
+
+def unknown_directive_arg(arg_name, directive_name, line, column):
+    return {
+        'message': KnownDirectives.message(arg_name, directive_name),
```

Comment:
Please change this to `KnownArgumentNames.unknown_directive_message(...)`. KnownDirectives rule has different error message.